Resources/Prototypes/_NF/Maps/Outpost/frontier.yml

```diff
@@ -19,11 +19,7 @@
           availableJobs:
             Passenger: [ -1, -1 ]
             HeadOfPersonnel: [ 1, 1 ]
-            HeadOfSecurity: [ 1, 1 ]
-            SeniorOfficer: [ 0, 0 ]
-            Brigmedic: [ 0, 0 ]
-            SecurityOfficer: [ 0, 0 ]
```

Comment:
keep the basic security officer in here for now, until we get the actual station guard role set up properly